lib/error/error_handler.rb

```diff
@@ -0,0 +1,98 @@
+module Error
+  module ErrorHandler
+    def self.included(clazz)
+      clazz.class_eval do
+        rescue_from ActionController::RoutingError do |error|
+          respond(:routing_error, 404, error.to_s)
+        end
```

Comment:
can it be in this block? { }

Reply:
we have a rubokop rule about the length of the line